plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/AnnotationManagerImpl.kt

```diff
@@ -120,6 +127,13 @@ abstract class AnnotationManagerImpl<G : Geometry, T : Annotation<G>, S : Annota
     gesturesPlugin.addOnMapClickListener(mapClickResolver)
     gesturesPlugin.addOnMapLongClickListener(mapLongClickResolver)
     gesturesPlugin.addOnMoveListener(mapMoveResolver)
+    if (workerThread == null) {
+      workerThread = HandlerThread("ANNOTATION_WORKER").apply {
+        priority = Thread.MAX_PRIORITY
+        start()
+        workerHandler = Handler(looper)
+      }
+    }
```

Comment:
nit: could workThread be lazily initialised?

Reply:
No, we will stop the thread in cleanup and need to recreate it when opening a map view again.